CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## Version 2.0.1
+
+*July 19, 2021*
+
+Documentation inconsistency between REST Api service and implementation was fixed:
+* in user_trades service documentation, now 'book' is not required.
```

Comment:
Does this correctly render as list? I've seen some list in API doc that doesn't look good.

Reply:
yes